src/components/Property/PropertyFlags/index.js

```diff
@@ -47,12 +47,10 @@ const PropertyFlags = ({
       .finally(() => setLocationAlertsLoading(false))
   }
 
-  const getPersonAlerts = (tenancyAgreementReference) => {
+  const getPersonAlerts = (tenureId) => {
     frontEndApiRequest({
       method: 'get',
-      path: `/api/properties/${encodeURIComponent(
-        tenancyAgreementReference
-      )}/person-alerts`,
+      path: `/api/properties/${tenureId}/person-alerts`,
```

Comment:
Would it not be personId instead of tenureId - I've never heard of an alert against a tenure